### PR TITLE
Fix display of Next Episode button

### LINKED ISSUE #1188: Next Episode Button not working and 30 second time is hardcoded
## Software Versions

- Jellyfin Server Version:  10.8.9
- Roku Client Version:  1.6.4

## Describe the bug
In JFVideo.brs, inside the sub checkTimeToDisplayNextEpisode() the Next Episode Button Time has been reverted back to a hardcoded 30 seconds. This should be using the `m.nextupbuttonseconds` variable.

When playing an episode the button never appeared like I expected.

## How To Reproduce
1. Play an episode of a TV show
2. Scroll to the end of the episode

## Expected behavior
The Next Episode button should appear either at 30 seconds, or at the time I set in my user settings.

### PR DESCRIPTION
**Pull request by 1hitsong**
## Changes
Updates code to use next up button seconds user setting when displaying Next Episode button.

Updates code to use video duration, which is build into Roku's video component, instead of custom runTime property, which isn't working properly.

## Issues
Fixes #1188

### CODE REVIEW

File: components/JFVideo.brs [hunk 1]
```diff
@@ -118,8 +122,9 @@ end sub
 ' Checks if we need to display the Next Episode button
 sub checkTimeToDisplayNextEpisode()
     if m.top.content.contenttype <> 4 then return
```
Comment (cewert) [inline components/JFVideo.brs:124]: This line was removed in `showNextEpisodeButton()` should it be removed here too?

File: components/JFVideo.brs [hunk 2]
```diff
@@ -100,7 +104,7 @@ end sub
 '
 'Update count down text
 sub updateCount()
-    nextEpisodeCountdown = Int(m.top.runTime - m.top.position)
+    nextEpisodeCountdown = Int(m.top.duration - m.top.position)
```
Comment (cewert) [inline components/JFVideo.brs:107]: why the change from runTime to duration?

Comment (1hitsong) [reply components/JFVideo.brs:107]: runTime is/was a custom property. Duration is a native property of the Video component.

File: components/JFVideo.brs (hunk 1 above)
Comment (1hitsong) [reply components/JFVideo.brs:124]: No, we want it, but we were checking the same thing twice, so I removed the 2nd one and left just the check at the top of the logic tree.

File: components/JFVideo.brs (hunk 2 above)
Comment (cewert) [reply components/JFVideo.brs:107]: Right but why make the change? is runTime less reliable or being deprecated or something?

Comment (1hitsong) [reply components/JFVideo.brs:107]: Oh, gotcha. It's only used in this one spot and it's a duplicate of an existing value that is automatically set. I'll open an issue to remove the code that sets this value.

**Review by cewert — APPROVED** [review]
